src/client/components/DatePicker/DatePicker.less

```diff
@@ -4,14 +4,26 @@
 }
 
 .opuscapita_date-picker__picker-container {
-  display: flex;
   background: #fff;
   box-shadow: 0 2px 6px rgba(0, 0, 0, 0.35);
   overflow: hidden;
   position: absolute;
   z-index: 9999;
+
+  left: 0;
+  top: 40px;
```

Comment:
Please, don't hardcode `top: 40px` (input height).

It can be solved in another way:

```
bottom: -4px;
transform: translate(0, 100%);
```